includes/class-wc-payments-woopay-button-handler.php

```diff
@@ -146,6 +146,12 @@ public function scripts() {
 		}
 
 		WC_Payments::register_script_with_dependencies( 'WCPAY_WOOPAY_EXPRESS_BUTTON', 'dist/woopay-express-button' );
+		wp_enqueue_style(
+			'WCPAY_WOOPAY_EXPRESS_BUTTON',
+			plugins_url( 'dist/woopay-express-button.css', WCPAY_PLUGIN_FILE ),
+			[],
+			'1.0'
+		);
```

Comment:
🤏 Nit: to keep consistency in this PR, as we applied https://github.com/Automattic/woocommerce-payments/pull/6985#discussion_r1294082482, we should update this as well:

```suggestion
		WC_Payments_Utils::register_style(
			'WCPAY_WOOPAY_EXPRESS_BUTTON',
			plugins_url( 'dist/woopay-express-button.css', WCPAY_PLUGIN_FILE ),
			[],
			WC_Payments::get_file_version( 'dist/woopay-express-button.css' )
		);
```

Reply:
Thanks for this! I updated this, but then found that the CSS wasn't being loaded then. I ended up using `WC_Payments_Utils::enqueue_style` and it puts the styles on the page. I was wondering if you could take a look at that commit and make sure what I did use the desired pattern there?

Reply:
I tested it and it happened the same to me 🤔 I find it weird that it worked in the aforementioned case. Do you think it'd be safer to replace `register_style` with `enqueue_style` in the other case too? I noted that the enqueue helper registers the style as well.

Reply:
As we discussed on our call, I updated the other `register_style` to also use `enqueue-style` as we discovered the discrepancy was between having different configurations of WooPay and Payment Request options enabled. I think we should do some additional testing around those different configurations now as well to confirm.

Reply:
Thanks, Brian! I played with all scenarios we can have with WooPay and Apple/Google Pay enabled/disabled and everything is looking as it should. Marking this as resolved 👍